nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -380,7 +388,19 @@ synchronized void initializeServer(final ProcessContext context) throws Exceptio
         final String clientAuthValue = context.getProperty(CLIENT_AUTH).getValue();
         final Server server = createServer(context);
 
-        final StandardServerConnectorFactory serverConnectorFactory = new StandardServerConnectorFactory(server, port);
+        NiFiProperties nifiProperties = NiFiProperties.createBasicNiFiProperties(null);
```

Comment:
The `NiFiProperties` class should never be referenced in Processors or extension components, so this changed be changed.

Reply:
This is a fair point and I will make the changes.

My motivation was simply trying to avoid property creep rather than taking a short-cut.  You may (or may not!) be surprised how many users think nifi.properties also applies to Processors.

Reply:
Thanks @mosermw! I certainly appreciate not wanting to introduce more properties than necessary.

I can also understand the confusion at times, particularly when it comes to HTTP and TLS properties. Thanks for working on the changes.

Reply:
Added a new commit that uses processor properties and moves some of the work into StandardServerConnectorFactory.